### PR TITLE
Search delete front end

### PR DESCRIPTION
**Pull request by sruthij1**

### CODE REVIEW

File: rentmyjunk/src/App.js [hunk 1]
```diff
@@ -80,21 +89,36 @@ function getData() {
...
+        {myStatus}
         <br />
         Woah
       </div>
     </div>
   );
 }
 
+function Search() {
+  return (
```
[the remark this answers is not in the document]
Comment (sruthij1) [reply rentmyjunk/src/App.js]: SearchForm.js is in the components folder, not in src. So I am not sure how to import it on the App.js page ?

[the remark this answers is not in the document]
Comment (N3ROO) [reply rentmyjunk/src/App.js]: You can add `import SearchForm from "./components/SearchForm"` at the top of the file, before `var local_host_url = blabla`

[the remark this answers is not in the document]
Comment (N3ROO) [reply rentmyjunk/src/App.js]: `./` refers to `/rentmyjunk/src` :)

File: components/SearchForm.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Form from './Form'
+
+class SearchForm extends Form {
+    render() {
+        return (
+            <div>
+              <h2>Search</h2>
+              <form>
```
[the remark this answers is not in the document]
Comment (sruthij1) [reply components/SearchForm.js]: Done :)

File: rentmyjunk/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sruthij1) [reply rentmyjunk/src/App.js]: Done :)

**Comment by N3ROO** [comment]
Perfect!

**Comment by N3ROO** [comment]
I think that we can merge this branch. SearchPost form has been updated to do multiple requests to the API if needed. The results are ignored for the moment because I can't do anything since the DB is empty. Notice that server.py is also empty in this branch.

Once that this branch is merged we will need to populate the DB to know how to handle the results.

**Pull request merged**